main.py

```diff
@@ -27,8 +24,10 @@ def webhook(request):
                 processed = handle_payload(jsonData)
 
                 if processed is False:
+                    logger.error("[ERROR] Error processing the payload")
                     return jsonify(message="Something went wrong!"), 400
                 elif processed == -1:
+                    logger.warning("[ERROR] Contact not found in the payload")
```

Comment:
Not use this if we are not using it on other logs
```suggestion
                    logger.warning("Contact not found in the payload")
```